src/main/java/uk/gov/hmcts/darts/common/repository/EventTypeRepository.java

```diff
@@ -9,5 +9,5 @@
 @Repository
 public interface EventTypeRepository extends JpaRepository<EventHandlerEntity, Integer> {
```

Comment:
I guess this Repository needs to be renamed following the change in table / entity name - `EventHandlerRepository`?

Reply:
done